terraform/api/aws/variables.tf

```diff
@@ -100,3 +100,8 @@ variable "subnets" {
 variable "vpc_id" {
   type = string
 }
+
+variable "pdb_default_min_available_percentage" {
```

Comment:
same here make it lexicographically sorted

Reply:
Sorted the variable and other ones too lexicographically.